src/framework/mlt_frame.c

```diff
@@ -470,7 +472,7 @@ int mlt_image_format_size( mlt_image_format format, int width, int height, int *
 			return 4;
 		case mlt_image_yuv422p16:
 			if ( bpp ) *bpp = 0;
-			return 4 * height * width ;
+			return height * ALIGN_TO( width * 2, 32 ) + 2 * height * ALIGN_TO( width, 32 ) ;
```

Comment:
Making all formats aligned will broke definitely existing pixel operation. New *yuv422p16* has no serious pixel processing routines except field-shift that works with strides properly...